### PR TITLE
Create new repository for Ansible BlueChi collection

### PR DESCRIPTION
**Pull request by ygalblum**

### CODE REVIEW

**Comment by netomi** [comment]
LGTM, can we get some approval from a project lead or fellow committer as this is your first request? Usually we try to learn from each project how they wanna do approve stuff like that.

**Comment by ygalblum** [comment]
@engelmi @mwperina PTAL

File: otterdog/eclipse-bluechi.jsonnet
Comment (mwperina) [inline otterdog/eclipse-bluechi.jsonnet]: Just wondering a name, wouldn't `bluechi-ansible-collection` be a better name?

Comment (ygalblum) [reply otterdog/eclipse-bluechi.jsonnet]: Sure, np. Done

**Review by mwperina — APPROVED** [review]
LGTM

**Review by netomi — APPROVED** [review]

**Comment by netomi** [comment]
changes are live, permissions for new repos might take up to 2h to be setup correctly as this is handled by a different service.

**Pull request merged**
